features/users/create_users.feature

```diff
@@ -47,9 +47,11 @@ Feature: As a developer
   Scenario: User signs up with a GitHub account having no public email (sad path)
     Given I am on the "registration" page
     When I click "GitHub"
-    Then I should see "Your Github account needs to have a public email address for sign up"
+    Then I should see "Your Github account needs to have a public email address for sign up. Click here for instructions."
     Then I should see the "github-alt" icon
     And I should not see "Password can't be blank"
+    And I click the "Click here for instructions" link
+    Then I should be on ""
```

Comment:
you don't necessarily have to have the test click through the link to the github page.  VCR could cover that for us by storing the request and then you could check for some relevant contents in the page.

Maybe more importantly we'd just like to be checking for a valid link ...